### PR TITLE
Changement = ajout sortie

### PR DESCRIPTION
**Pull request by mdan312**

### CODE REVIEW

**Comment by mdan312** [comment]
@casey-collab

File: french/manger/sortie.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Ça fait du bien de pouvoir respirer le grand air!!!
```
Comment (casey-collab) [inline french/manger/sortie.md]: It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by mdan312** [comment]
Ok

**Comment by mdan312** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :purple_heart:

**Pull request merged**